### DISCUSSION TITLE
Semantic Kernel image-to-text models

### DISCUSSION THREAD

**Discussion by AngelStubbings** [microsoft/semantic-kernel#7447] [General]
Regarding this [blog post](https://devblogs.microsoft.com/semantic-kernel/image-to-text-with-semantic-kernel-and-huggingface/), can you use any image-to-text models such as `Salesforce/xgen-mm-phi3-mini-instruct-r-v1` or does it have to be `Salesforce/blip-image-captioning-base` like in the demo?

**Comment by rogerbarreto** [comment]
@AngelStubbings , thanks for the question, the example should work with any Image-To-Text model from HuggingFace assuming they are respecting the contract from the Hugging Face APIs, let me know if that works and feel free to reopen this issue if not. Thanks!
